pkg/pgmodel/migrations/sql/1_base_schema.up.sql

```diff
@@ -197,7 +197,8 @@ BEGIN
     -- stupidly create table ... partition of X first takes an access share lock
     -- and then an access exclusive lock. Such an upgrade can, and does, cause
     -- deadlocks. Taking the stronger lock beforehand prevents such deadlocks.
-    LOCK TABLE _prom_catalog.series in ACCESS EXCLUSIVE mode;
+    -- We only need the lock on the parent.
+    LOCK TABLE ONLY SCHEMA_CATALOG.series in ACCESS EXCLUSIVE mode;
```

Comment:
Got it, we're probably still access share locking hte other children to prevent their constraints from being changed at the same time. But that's fine.